docs/guides/typescript.md

```diff
@@ -318,15 +323,14 @@ const useBearStore = create<BearState>()(
 ```ts
 import {
   create,
-  State,
```

Comment:
A working example can be found here: https://stackblitz.com/edit/vitejs-vite-zm1jpg?file=src%2FApp.tsx